resfulservice/src/controllers/kgWrapperController.js

```diff
@@ -218,6 +218,29 @@ exports.getAllCharts = async (req, res, next) => {
   }
 };
 
+/**
+ * Load dataset gallery from elastic function
+ * @param {*} req
+ * @param {*} res
+ * @param {*} next
+ * @returns {*} response
+ */
+exports.getAllDatasets = async (req, res, next) => {
+  const page = parseInt(req?.query?.page) || 1;
+  const pageSize = parseInt(req?.query?.pageSize) || 10;
+
+  try {
+    const response = await elasticSearch.loadAllDatasets(page, pageSize);
```

Comment:
This returns a 404 error. After investigation, datasets index doesn't exist yet in elastic search or initialized.